src/cargo/ops/cargo_new.rs

```diff
@@ -258,6 +258,13 @@ fn check_name(
             name
         ))?;
     }
+    let name_in_lowercase = name.to_lowercase();
+    if name != name_in_lowercase {
+        shell.warn(format!(
+            "package `{}` should have a lowercase name such as `{}`",
+            name, name_in_lowercase
```

Comment:
nit: saying this just because I already made another comment but feel free to reference these inside the `format!` string (`format!("package '{name}'...`)

Reply:
There are few more places in the method where this applies.  I'll send a separate PR addressing these.

Reply:
Yup, not accountable for fixing everything but we can improve what new code we write :)